doc/tutorials/iOS_tutorial.md

```diff
@@ -413,8 +413,8 @@ extension ViewController: LogInViewControllerDelegate {
 }
 ```
 
-Why are we adding `ViewController` as a delegate of `XMPPStream` if our `XMPPController` alreay has that delegate implemented?
-Because we need to know if this connection and authentication was successfull or notin our `ViewController`
+Why are we adding `ViewController` as a delegate of `XMPPStream` if our `XMPPController` already has that delegate implemented?
+Because we need to know if this connection and authentication was successful or notin our `ViewController`
```

Comment:
Maybe fix this one as well?
```suggestion
Because we need to know if this connection and authentication was successful or not in our `ViewController`
```